react-components/package.json

```diff
@@ -20,7 +20,7 @@
         "prop-types": "^15.6.2"
     },
     "dependencies": {
-        "@cyverse-de/ui-lib": "0.3.6",
+        "@cyverse-de/ui-lib": "file:../../ui-lib/cyverse-de-ui-lib-0.3.6.tgz",
```

Comment:
Just a reminder to switch this back to the official `ui-lib` version before merging 😉

Reply:
✔️